components/climber.py

```diff
@@ -1,13 +1,45 @@
+import wpilib
+from rev import CANSparkMax
+from ids import SparkMaxIds
+
+
 class ClimberComponent:
     def __init__(self) -> None:
         self.deployed = False
 
     def deploy(self) -> None:
         self.deployed = True
 
+    def execute(self) -> None:
+        if self.deployed:
+
+            def __init__(self) -> None:
+                self.deployed = False
+
+    def deploy(self) -> None:
+        self.deployed = True
+
     def execute(self) -> None:
         if self.deployed:
             # Deploy the climber
-            pass
+            self.climbingmotor = CANSparkMax(
+                SparkMaxIds.climber, CANSparkMax.MotorType.kBrushless
+            )  # using a Neo with 4:1 4:1 3:1 ratio
+            self.limitswitch = wpilib.LimitSwitch(2)
+
+            def motorCurrent():
+                return CANSparkMax.getOutputCurrent
```

Comment:
return the return value for the method of getOutputCurrent used on the motor you added right. so in this case, I think you want to call the method on your self.climbingmotor

Reply:
You also need the brackets to actually make the function call e.g. self.climbingmotor.getOutputCurrent()